.vscode/launch.json

```diff
@@ -2,38 +2,20 @@
   "version": "0.2.0",
   "configurations": [
     {
-      "name": "Next.js: Server",
-      "type": "node-terminal",
+      "type": "node",
       "request": "launch",
-      "command": "npm run dev",
-      "skipFiles": ["<node_internals>/**"],
-      "outFiles": [
-        "${workspaceFolder}/**/*.js",
-        "!**/node_modules/**"
```

Comment:
Is this file meant to be changed in this PR?

Reply:
Oh you are right, although we can leave it, couldn't get the debugger work without these changes.